rgw/v2/tests/s3_swift/reusable.py

```diff
@@ -1596,11 +1598,19 @@ def pipe_operation(group_id, pipe_op, zone_names=None, bucket_name=None):
         bkt = f" --bucket={bucket_name}"
     else:
         bkt = ""
-    cmd = (
-        f"radosgw-admin sync group pipe {pipe_op} --group-id={group_id} --pipe-id={pipe_id}"
-        + zn
-        + bkt
-    )
+    if policy_detail is not None:
+        cmd = (
+            f"radosgw-admin sync group pipe {pipe_op} --group-id={group_id} --pipe-id={pipe_id}"
+            + zn
+            + bkt
+            + policy_detail
+        )
+    else:
+        cmd = (
+            f"radosgw-admin sync group pipe {pipe_op} --group-id={group_id} --pipe-id={pipe_id}"
+            + zn
+            + bkt
+        )
```

Comment:
IMO , this can be used as
cmd =  f"radosgw-admin sync group pipe {pipe_op} --group-id={group_id} --pipe-id={pipe_id}" + zn + bkt
if policy_detail is not None:
        cmd=cmd + policy_detail

Reply:
thank you @anrao19 , updated, pls review.